src/components/text-area/text-area.component.js

```diff
@@ -1,7 +1,8 @@
 const TextArea = ({ ...props }) => (
   <textarea
     {...props}
-    name="boxContent"
+    name="content"
+    id='text-area'
```

Comment:
You can leave this for now but ideally it's not a good practice to add `id`s to reusable components.
If you had two textareas on the same page, you'd have two elements with the same id but that shouldn't be so https://www.w3schools.com/html/html_id.asp

Reply:
I found a way to successfully trigger the focus in the box component.
The id **'text-area'** was changed under this effect


> in [src/components/box/box.component.js](https://github.com/Seun-A/weekly-meal-planner/pull/12/files/39556b29107681b509f19f2f554f78eb20732038#diff-9e48d92febadff0dcaa42367db7a519b28067a23f0399784535eaa6535fbc2b4)

` setTimeout(() => { document.getElementById('edit-text-area').focus() }, 170); `

I plan to modify this later